examples/misc_instancing.html

```diff
@@ -0,0 +1,231 @@
+<html>
+    <head>
+        <title>Itowns - Instancing</title>
+        <meta charset="UTF-8" />
+        <link rel="stylesheet" type="text/css" href="css/example.css" />
+        <link rel="stylesheet" type="text/css" href="css/LoadingScreen.css" />
+
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <script src="https://cdnjs.cloudflare.com/ajax/libs/dat-gui/0.7.6/dat.gui.min.js"></script>
+    </head>
+
+    <body style="background-color: black">
+        <div id="description">
+            <p>
+                <b
+                    >Render a large number of objects with the same geometry<br />
```

Comment:
Can you remove the `<br />` markup which has no effect here ?

Reply:
I think it looks better with a line break:
![Screenshot from 2023-06-16 09-59-21](https://github.com/iTowns/itowns/assets/126568810/f981eb7a-9758-4633-80b6-01a526a16491)
![Screenshot from 2023-06-16 09-59-33](https://github.com/iTowns/itowns/assets/126568810/93ee0f54-5128-453d-87af-93cf6f325965)

Reply:
Indeed, let's keep it that way. As I checked it with the navigator console opened, the width of the description was reduced and the line breaking was forced (by the `max-width: 40%;` in css) so I did not see a difference.